service/src/main/java/bio/terra/cbas/initialization/InstanceInitializerBeanConfig.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.cbas.initialization;
+
+import bio.terra.cbas.config.CbasContextConfiguration;
+import bio.terra.cbas.dao.MethodDao;
+import bio.terra.cbas.dao.MethodVersionDao;
+import bio.terra.cbas.dao.RunDao;
+import bio.terra.cbas.dao.RunSetDao;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class InstanceInitializerBeanConfig {
+
+  @Bean
+  public CloneRecoveryBean cloneRecoveryBean(
+      RunSetDao runSetDao,
+      RunDao runDao,
```

Comment:
since we don't update anything in run table do we need this here?

Reply:
ah, good catch. you're right!